components/network/pdns-recursor/patches/00-wontbuild.patch

```diff
@@ -0,0 +1,101 @@
+--- pdns-recursor-4.3.0/ext/yahttp/yahttp/reqresp.cpp.orig	2020-05-17 13:22:48.488029417 +0000
```

Comment:
Can you insert link to https://github.com/PowerDNS/pdns/pull/9031 ?

Reply:
Done and I added the testsuite, all tests passed.